docs/production-deployment/cloud/operation-api.mdx

```diff
@@ -18,38 +18,49 @@ The Temporal Cloud Operations API is in [Public Preview](/evaluate/development-p
+The Temporal Cloud Operations API, or the Cloud Ops API, is an open source, public [HTTP API](https://rest.saas-api.tmprl.cloud/docs/httpapi.html#description/introduction) and [gRPC API](https://github.com/temporalio/cloud-api/tree/main) for programmatically managing Temporal Cloud control plane resources, including [Namespaces](/cloud/namespaces), [Users](/cloud/users), [Service Accounts](/cloud/service-accounts), [API keys](/cloud/api-keys), and others. The Temporal Cloud [Terraform Provider](/production-deployment/cloud/terraform-provider), [tcld CLI](/cloud/tcld), and Web UI all use the Cloud Ops API.
 
-[Full docs are available on Buf](https://buf.build/temporalio/cloud-api/docs/main:temporal.api.cloud.cloudservice.v1#temporal.api.cloud.cloudservice.v1.CloudService).
+## Develop applications with the Cloud Ops API
 
-You can use the provided proto files to generate client libraries in your desired programming language, and then use that client to access the API.
+You can use the HTTP API or the gRPC API depending on how you need to integrate with your platform.
 
-The Temporal Cloud [Terraform Provider](/production-deployment/cloud/terraform-provider), [tcld CLI](/cloud/tcld), and Web UI all use the Cloud Ops API.
+### Prerequisites
 
-## SDKs and samples
+These prerequisites are required for using either HTTP or gRPC.
 
-We provide a [Go SDK](https://github.com/temporalio/cloud-sdk-go) that compiles the proto files into Go bindings, offering a more idiomatic interface for Go developers. This SDK eliminates the need to manually compile protobuf files when working with Go.
+- [Temporal Cloud user account](/cloud/get-started)
+- [API Key](/cloud/tcld/apikey#create) for authentication
 
-We also provide [Go samples](https://github.com/temporalio/cloud-samples-go) to help you get started with the Cloud Ops API using the Go SDK.
+### Use cases
 
-## Develop applications with the Cloud Ops API
+Some common reasons you might use the API are to:
 
-### Prerequisites
+- Provision Namespaces per environment or tenant via pipelines.
+- Bootstrap new projects by creating users, assigning roles, and creating Namespaces via custom scripts.
+- Rotate service account keys on a schedule with a job.
+- Audit and report access across orgs with scheduled HTTP requests.
 
-- [Temporal Cloud user account](/cloud/get-started)
-- [API Key](/cloud/tcld/apikey#create) for authentication
+### Using HTTP
+
+[The HTTP API](https://rest.saas-api.tmprl.cloud/docs/httpapi.html#description/introduction) supports the same operations as the [gRPC API](#using-grpc), but it's usable via standard HTTP methods and authentication. This may be a more convenient option if you are writing automation scripts for CI/CD or you can't use gRPC due to network policies, proxies, tooling gaps, or language/runtime constraints. Since it's standard HTTP, it's language agnostic giving you the ability to run cloud operations consistently.
```

Comment:
```suggestion
[The HTTP API](https://saas-api.tmprl.cloud/docs/httpapi.html#description/introduction) supports the same operations as the [gRPC API](#using-grpc), but it's usable via standard HTTP methods and authentication. This may be a more convenient option if you are writing automation scripts for CI/CD or you can't use gRPC due to network policies, proxies, tooling gaps, or language/runtime constraints. Since it's standard HTTP, it's language agnostic giving you the ability to run cloud operations consistently.
```